lib/init/config-initializer.js

```diff
@@ -391,7 +394,24 @@ function promptUser() {
             message: "Does your project use TypeScript?",
             enabled: "Yes",
             disabled: "No",
-            initial: 0
+            initial: 0,
+            result(val) {
```

Comment:
I tried this:

```
√ How would you like to use ESLint? · problems
√ What type of modules does your project use? · esm
√ Which framework does your project use? · none
√ Does your project use TypeScript? · No / Yes
√ Where does your code run? · browser
√ What format do you want your config file to be in? · JavaScript
Local ESLint installation not found.
The config that you've selected requires the following dependencies:

eslint@latest
√ Would you like to install them now? · No / Yes
A config file was generated, but the config file itself may not follow your linting rules.
Successfully created .eslintrc.js file in D:\tmp\tmp
```

(TypeScript was Yes)

And got this config:

```js
module.exports = {
    "env": {
        "browser": true,
        "es2021": true
    },
    "extends": "eslint:recommended",
    "parserOptions": {
        "ecmaVersion": "latest",
        "sourceType": "module"
    },
    "rules": {
    }
}

```

There is nothing related to TypeScript in the config.

I think what's missing in this function is `return val;`. Otherwise, it implicitly returns `undefined` which overwrites `answers.typescript` with `false`.

Reply:
ah, my bad! fixed in c21f4c2ffef6c58e6001912e53bc4ad282d115b1

great to having you as a reviewer! ❤️